packages/optimistic-oracle/index.js

```diff
@@ -119,8 +129,14 @@ async function Poll(callback) {
       errorRetries: process.env.ERROR_RETRIES ? Number(process.env.ERROR_RETRIES) : 5,
       // Default to 10 seconds in between error re-tries.
       errorRetriesTimeout: process.env.ERROR_RETRIES_TIMEOUT ? Number(process.env.ERROR_RETRIES_TIMEOUT) : 10,
+      // Common price feed configuration passed along to all those constructed by proposer.
+      commonPriceFeedConfig: process.env.OOPRICE_FEED_CONFIG
+        ? JSON.parse(process.env.OOPRICE_FEED_CONFIG)
+        : { lookback: 7200 },
       // If there is an optimistic oracle config, add it. Else, set to null. Example config:
       // {
+      //   "disputePriceErrorPercent":0.05 -> Proposal prices that do not equal the dispute price
```

Comment:
Does this mean 0.05% or 5%? Looks like 5% from reading the code, but the comment could be more explicit.

Reply:
5%, clarified in the code.